v2/src/lib.rs

```diff
@@ -611,7 +625,7 @@ impl Storage for InfluxDbStorage {
                                             |> filter(fn: (r) => r._measurement == \"{}\")
```

Comment:
I think the strict equality won't work in case the queried keyexpr contains `*` or `**`.
With InfluxDB v1 the keyexpr is converted to an InfluxQL regular epxression [here](https://github.com/eclipse-zenoh/zenoh-backend-influxdb/blob/bedc328ffb894abfaf1ebc9724869249ec7c2d0f/v1/src/lib.rs#L512) and this regex is used in the query, instead of the keyexpr.
The same shall be done for v2, using [Flux regular expressions](https://docs.influxdata.com/flux/v0/get-started/syntax-basics/#regular-expression-types) in fliter predicate.

Reply:
I have implemented mapping Key Expressions with Wildcards to regular expressions.
However i've upon testing, ive found running `z_get.py`, the `get()` function is called individually with each Key Expr output from `get_all`

![image](https://github.com/user-attachments/assets/5ec080f4-2343-47d3-b99d-7a8afaa66670)

@JEnoch 
Is there a way to call this such that the `get` gets called with a wildcard KeyExpr ?